deepchecks/nlp/utils/text_properties.py

```diff
@@ -265,6 +269,11 @@ def average_word_length(text: str) -> float:
 
 def percentage_special_characters(text: str) -> float:
     """Return percentage of special characters (as float between 0 and 1)."""
+    return len([c for c in text if c in NON_PUNCTUATION_SPECIAL_CHARS]) / len(text) if len(text) != 0 else 0
```

Comment:
What about what we said about non-letter/digits?

Reply:
Can you remind me? What did we say?